plugins/ImageReader/ImageReader.py

```diff
@@ -99,12 +101,14 @@ def _generateSceneNode(self, file_name, xz_size, peak_height, base_height, blur_
         for x in range(0, width):
             for y in range(0, height):
                 qrgb = img.pixel(x, y)
-                avg = float(qRed(qrgb) + qGreen(qrgb) + qBlue(qrgb)) / (3 * 255)
-                height_data[y, x] = avg
+                if use_transparency_model:
+                    height_data[y, x] = (0.299 * math.pow(qRed(qrgb) / 255.0, 2.2) + 0.587 * math.pow(qGreen(qrgb) / 255.0, 2.2) + 0.114 * math.pow(qBlue(qrgb) / 255.0, 2.2))
+                else:
+                    height_data[y, x] = (0.212655 * qRed(qrgb) + 0.715158 * qGreen(qrgb) + 0.072187 * qBlue(qrgb)) / 255 # fast computation ignoring gamma and degamma
 
         Job.yieldThread()
 
-        if not lighter_is_higher:
+        if lighter_is_higher is use_transparency_model:
```

Comment:
use == for equality, `is` for identity checks

Reply:
ok fixed